Sources/Crypto/RSA/RSAKey.swift

```diff
@@ -102,6 +107,33 @@ final class CRSAKey {
         self.pointer = pointer
     }
 
+    /// Creates a new DER encoded `CRSAKey` from data.
+    static func make(from der: Data) throws -> CRSAKey {
+        guard let derDecoded = Data(base64URLEncoded: der) else {
```

Comment:
Is being base-64-url encoded part of the DER specification? Or is that specific to DER inside of a JWT?
If this is just a JWT thing, we should expect the user to base-64-url decode it before passing the data to this method.

Reply:
I thought so, but it's not.